versioned_docs/version-1.29/reference/CRDs.md

```diff
@@ -0,0 +1,649 @@
+| `sideEffects` _[SideEffectClass](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.31/#sideeffectclass-v1-admissionregistration)_         | SideEffects states whether this webhook has side effects.<br />Acceptable values are: None, NoneOnDryRun (webhooks created via v1beta1 may also specify Some or Unknown).<br />Webhooks with side effects MUST implement a reconciliation system, since a request may be<br />rejected by a future step in the admission change and the side effects therefore need to be undone.<br />Requests with the dryRun attribute will be auto-rejected if they match a webhook with<br />sideEffects == Unknown or Some.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |         |                                |
+| `timeoutSeconds` _integer_                                                                                                                               | TimeoutSeconds specifies the timeout for this webhook. After the timeout passes,<br />the webhook call will be ignored or the API call will fail based on the<br />failure policy.<br />The timeout value must be between 1 and 30 seconds.<br />Default to 10 seconds.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | 10      |                                |
```

Comment:
Argh, `timeoutEvalSeconds` is not consumed even after retriggering the job to recreate this PR.

I suppose because we incorrectly released kubewarden-controller v1.29.0 with the CRD changes but without the docs changes.

@jhkrug included the changes into the kubewarden-controller with https://github.com/kubewarden/kubewarden-controller/pull/1219.

We may need to import this by hand into the docs, prior to triggering this PR.